reactor-netty-core/src/main/java/reactor/netty/FutureMono.java

```diff
@@ -80,21 +80,8 @@
 
 		@Override
 		@SuppressWarnings("FutureReturnValueIgnored")
```

Comment:
nit: remove the suppression above @SuppressWarnings("FutureReturnValueIgnored")

it is related to `addListener` invocation

Reply:
done